volume/volume.go

```diff
@@ -154,7 +154,7 @@ func (m *MountPoint) Type() string {
 // ParseVolumesFrom ensures that the supplied volumes-from is valid.
 func ParseVolumesFrom(spec string) (string, string, error) {
 	if len(spec) == 0 {
-		return "", "", fmt.Errorf("malformed volumes-from specification: %s", spec)
+		return "", "", fmt.Errorf("Volumes-from specification can not be an empty string!")
```

Comment:
can you; remove the exclamation mark (`!`), change "can not" to a single word, and make `Volumes-from` lowercase? So;

    "volumes-from specification cannot be an empty string"